types/types.go

```diff
@@ -906,15 +906,36 @@ const (
 	PaysetCommitMerkle
 )
 
+// AssetParamsWithExtra adds byte arrays to store non UTF8 asset strings.
+type AssetParamsWithExtra struct {
```

Comment:
you are using this only for the database persistence, right ? if so, wouldn't it be better to move this to the database layer ? ( i.e. idb/* )

Reply:
If it's db-only then it's postgres-only, because hopefully another db won't have the problem with storing data that Postgres has which is the cause of all this runaround.

Reply:
sorry - I did not mean it needs to move *directly* under `idb`, but rather under the `idb` root.

Reply:
Thanks, I'll move it to postgres